src/js/lib/admin-page/components/Setting.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { SelectControl, TextControl, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { PSEUDO_BOOLEAN_SETTINGS } from '../constants';
+
+const Setting = ( { name, note, value, onChange, label } ) => {
+	const _onChange = ( newValue ) => onChange( [ name, newValue ] );
+	let input;
+
+	if ( PSEUDO_BOOLEAN_SETTINGS.includes( name ) ) {
+		input = <ToggleControl name={ name } label={ label } onChange={ _onChange } checked={ value } />;
+	} else if ( [ 'track_post_types', 'track_page_types' ].includes( name ) ) {
+		input = (
+			<SelectControl
```

Comment:
Multi-select elements are notoriously inaccessible - can we find a way to drop them completely?